drivers/ethernet/eth_sam_gmac.c

```diff
@@ -2222,8 +2222,8 @@ static const struct eth_sam_dev_cfg eth0_config = {
 	.clock_cfg = SAM_DT_INST_CLOCK_PMC_CFG(0),
 #endif
 	.config_func = eth0_irq_config,
-#if DT_NODE_EXISTS(DT_INST_CHILD(0, phy))
-	.phy_dev = DEVICE_DT_GET(DT_INST_CHILD(0, phy))
+#if DT_INST_NODE_HAS_PROP(0, phy_handle)
+	.phy_dev = DEVICE_DT_GET(DT_INST_PHANDLE(0, phy_handle))
 #else
 #error "No PHY driver specified"
```

Comment:
@nandojve hey you want me to make `phy-handle` required in the yaml file and drop this?

Reply:
Yes, please!

Reply:
done